### PR TITLE
[IMP] Email marketing: Analyze metrics reports

### PR DESCRIPTION
**Pull request by pabr-odoo**
Task card: https://www.odoo.com/odoo/project/3835/tasks/4112632

Overhauling the analyze metrics doc to refocus on specific use cases rather than just definitions and contain information on generating different types of reports using the analysis dashboard.

### CODE REVIEW

**Comment by pabr-odoo** [comment]
Hey @ksc-odoo have an improvement ready for first review from you whenever you get the chance. Thanks!

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:113]: Please correct me if I'm mistaken, but shouldn't the screenshot being used here show 6.9% in the 'Bounced' smart button?

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:119]: I think this example might benefit from some clarification.

When I click on 'Bounced' to follow this example, and I click 'Add Custom Group', it reveals a long list of options, but 'Mailing Lists' isn't one of them. Perhaps that's because, on the runbot I'm using for the review, there is only one contact appearing on the 'Bounced' page, making such a grouping impossible. Either way, I'd suggest confirming the presence of this during your revisions.

And, I'd suggest either ref-linking to a doc about how to create a custom filter, or providing the reader with a brief breakdown of how to create a custom filter, so they can be armed with that knowledge, should they want to create one of their own.
```suggestion
   To see what these bounced recipients have in common, the records are grouped using a custom group
   targeting :guilabel:`Mailing Lists`, which groups all records by the mailing lists they are on.
   The records are then filtered using a custom filter with the rule `Created on >= 07/01/2024
   00:00:00`, when the mailing list was last checked. This filter only includes recipients
   that have been created on, or after, July 1st, 2024, in the report.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:125]:
```suggestion
   Using these search configurations, it can be seen that all the recipients with bounced emails were added
```
optional rewording

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:126]:
```suggestion
   after the list was last checked. Looking closer at the domains, it is evident that each
```
optional rewording

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:140]:
```suggestion
Analysis* report. To begin, navigate to :menuselection:`Email Marketing app --> Reporting --> Mass
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (ksc-odoo) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:145]:
```suggestion
:guilabel:`Measures` button, and select the desired measure from the drop-down menu.
```

**Review by ksc-odoo — APPROVED** [review]
hey @pabr-odoo -- just finished my Peer Review of this PR. There are a decent amount of comments that require your attention, but once you address those, and implement the necessary feedback, feel free to move this to the next stage. if you have any questions, don't hesitate to reach out. Thanks! 👍

**Comment by pabr-odoo** [comment]
Hey @StraubCreative this improvement is ready for your final review whenever you get the chance. Thanks!

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (StraubCreative) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:103]: *Bounce* does not appear to be a readily available Group By option, nor is it listed in the **Add Custom Group** menu. Is this actually a selectable Group By?
image

Comment (StraubCreative) [reply content/applications/marketing/email_marketing/analyze_metrics.rst:103]: A more relevant Group By nesting example here, I'd argue, would be something like **Sent Period** > **Winner Selection** to see Email Marketing campaign "winners" over a specific time frame. 

**Sent Period** > **Responsible** would be another good one to see who on the team is sending in volume or quality (team performance check)

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (StraubCreative) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:109]: Good example 👍

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (StraubCreative) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:152]: GREAT example! 
Consider writing a tutorial just on this topic around open rate optimization 🤙

**Review by StraubCreative — APPROVED** [review]
Hi @pabr-odoo 

Good rewrite. I think this doc will set you nicely up to do more advanced reporting flows for Email Marketing.

The comments I have below are mostly around presentation or relevance, with one item in particular I'd like you to double-check for validity (*Bounced* as a filter/group by doesn't exist I don't think?). After your done addressing the comments, as you see fit, this will be ready for tech review, ty!

**Comment by pabr-odoo** [comment]
Hey @samueljlieber this PR is ready for tech review whenever you get the chance. Thanks!

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:81]: Remove slash :)
```suggestion
This rate measures the percentage of emails that were **successfully** delivered, against the total
number of sent emails.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:135]: Indent entire text to align with the admonition, also fix menuselection :)
```suggestion
   Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
   is selected first. For example, selecting :guilabel:`Sent Period`, followed by
   :guilabel:`Add Custom Group --> Responsible`, in the :guilabel:`Group By` column, sorts all
   results *first* by the sent period, *then* by the team member responsible. This is a useful
   metric for analyzing who on the team is sending in volume or quantity over a set time period.

   This can be verified by looking at the direction, and order, of the selections in the group tile
   that appears in the search bar after the selections are made.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:148]: Just double checking this statement, as it wasnt clear to me in this context:
> "when the mailing list was last checked."

```suggestion
   00:00:00`, to filter by when the mailing list was last checked. This filter only includes
   recipients that have been created on, or after, July 1st, 2024, in the report.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:152]: Scoot image back to align with the admonition
```suggestion
   .. image:: analyze_metrics/metrics-filter.png
      :align: center
      :alt: The custom filter creation form.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:165]: Indent text again :)
```suggestion
   View :doc:`../../essentials/search` for more information about making custom groups and
   filters.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:186]: Maintain consistent terminology :)
```suggestion
   In this view, it can be seen that the first mass mailing led to a higher opened rate than the
   second. Because a lower opened rate can sometimes be attributed to a subject line that failed to
   capture readers' attention, the subject line of each mass mailing can be a good place to begin
   looking.
```

File: content/applications/marketing/email_marketing/analyze_metrics.rst
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/analyze_metrics.rst:178]: This text can be seen on the current UI, so please use a guilabel :)
```suggestion
A dashboard appears displaying a bar chart containing each mailing campaign. By default,
:guilabel:`Sent` is selected, displaying the number of sent records on the y-axis. To change the
measure, click the :guilabel:`Measures` button, and select the desired measure from the drop-down
menu.
```

**Review by samueljlieber — APPROVED** [review]
Hey @pabr-odoo! Nice work on this improvement to the analyze metrics reports doc for email marketing! I am approving with a couple corrections and suggestions for you!

Please run `make review` to catch a few early line breaks :)

Thank you!
...
@robodoo delegate=pabr-odoo

**Comment by pabr-odoo** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
This PR's merge function appears to have malfunctioned?

Clean rebase on [331f11a](https://github.com/odoo/documentation/pull/10764/commits/331f11aa442a3a176f298a772f6537415b6f84fe) (no conflicts).
...
@robodoo r+

**Comment by samueljlieber** [comment]
@robodoo retry

**Comment by samueljlieber** [comment]
@robodoo retry

**Pull request closed without merging**
